docs/user_guide/HOWTO/Configure_Ssh.rst

```diff
@@ -1,118 +1,106 @@
 HOWTO Customize Settings for SSH
 ================================
 
-By default, including the **ssh** module will include ``ssh::server`` and
-``ssh::client``, configured with reasonable defaults for the OS and environment:
+By default, SIMP will include the ``simp-ssh`` module ``ssh`` class under all
+deployment scenarios. To exclude the SIMP ``ssh`` class, refer to
+:ref:`Disable SSH Managment <disable_ssh>`. To include specify the class for
+inclusion, add the following to :term:`Hiera`:
 
-.. code-block:: puppet
+.. code-block:: yaml
 
-   include 'ssh'
+   simp::classes:
+     - 'ssh'
 
-.. NOTE::
 
-   The examples below feature ``include 'ssh::server'`` and ``include
-   'ssh::client'``, but most SIMP scenarios already include them both via
-   ``ssh``.  So, for SIMP systems, you will customize parameter settings
-   for ``ssh::server`` and ``ssh::client`` via Hiera.
+The SIMP ``ssh`` class is configured to automatically include the
+``ssh::server`` and ``ssh::client`` classes. These classes manage the SSH
+daemon settings for incoming connections and the SSH client settings for
+outgoing connections respectively, and are configured with reasonable defaults
+for the OS and environment. To override this and **disable management** of one
+or both of these classes and manage them through some other mechanism, add the
+following to Hiera:
+
+.. code-block:: yaml
+
+   ssh::enable_client: false
+   ssh::enable_server: false
 
 
 Managing Settings for the SSH Server
 ------------------------------------
 
-Including ``ssh::server`` with the default options will manage the server with
-reasonable settings for each host's environment.
+As stated above, the ``simp-ssh`` module includes the ``ssh::server`` class and
+provides "sane" settings for each host's environment by default. Detailed
+descriptions of the various settings are provided in the sshd_config(5) man
+page.
 
 
 Configuring ``ssh::server::conf`` from Hiera
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To customize the SSH server, edit the parameters of ``ssh::server::conf`` using
-:term:`Hiera` or :term:`ENC`.
+Many standard sshd_config settings for the SSH server are included with default
+values in the ``ssh::server::conf`` class. To customize the SSH server, edit
```

Comment:
To better connect these two sentences, I suggest replacing `To customize the SSH server` with `To customize these settings`, or (clearer but more repetitive),  `To customize these sshd_config settings`.